libraries/chain/include/eosio/chain/unapplied_transaction_queue.hpp

```diff
@@ -255,7 +255,7 @@ class unapplied_transaction_queue {
 
    static uint64_t calc_size( const transaction_metadata_ptr& trx ) {
       // packed_trx caches unpacked transaction so double
-      return (trx->packed_trx()->get_unprunable_size() + trx->packed_trx()->get_prunable_size()) * 2 + sizeof( *trx );
+      return trx->packed_trx()->get_estimated_size() + sizeof( *trx );
```

Comment:
Isn't our estimate using `sizeof( *trx )` a little silly? We are also missing various fields, like the ones in the `unapplied_transaction` struct. Maybe it would be better to just add some const overhead.

Also `sizeof( *trx )` is not accounting for the size of the recovered public keys. I don't know if we care. If we do perhaps we should just add an `get_estimated_size()` to `transaction_metatdata_ptr` and use that (plus the fixed overhead in the `unapplied_transaction` struct).

Reply:
We don't really care to be precise, but seemed easier to just add the `get_estimated_size()` and use that.